carma/launch/plugins.launch.py

```diff
@@ -559,12 +559,12 @@ def generate_launch_description():
         namespace=GetCurrentNamespace(),
         composable_node_descriptions=[
             ComposableNode(
-                package='platoon_strategic_ihp',
-                plugin='platoon_strategic_ihp::Node',
-                name='platoon_strategic_ihp_node',
+                package='platooning_strategic_ihp',
```

Comment:
These changes should also be made in the package.xml and CMakeLists.txt of `platooning_strategic_ihp`

Reply:
Anish is handling this in his PR, so we can probably leave this for now